From 014880ef45b8798c44050bd26dcb50b4e133d729 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 29 Apr 2021 19:55:11 +0300
Subject: [PATCH] [SW-0] fwupd: print error message for unknown fit

When the load type of the FIT is not recognized, print an error message.

Change-Id: I17b27225a69476310a0a0457c315616be14daa25
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/preboot/module_fwupd.c b/preboot/module_fwupd.c
index dd3fff1..1dbb0fe 100644
--- a/preboot/module_fwupd.c
+++ b/preboot/module_fwupd.c
@@ -205,6 +205,7 @@
 	case FIT_LOAD_TYPE_RUN:
 		return module_fwupd_load_type_run();
 	default:
+		preboot_puts("Unknown FIT load type\n");
 		return 1;
 	}
 
